src/features/requests/components/data-table-row-actions.tsx

```
@@ -0,0 +1,53 @@
+                    <Button
+                        variant='ghost'
+                        className='data-[state=open]:bg-muted flex h-8 w-8 p-0'
+                    >
+                        <DotsHorizontalIcon className='h-4 w-4' />
+                        <span className='sr-only'>Mở menu</span>
+                    </Button>
+                </DropdownMenuTrigger>
+                <DropdownMenuContent align='end' className='w-[160px]'>
+                    <DropdownMenuItem
+                        onClick={() => {
+                            const { user } = useUser();
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)